tests/test_logging.py

```diff
@@ -193,6 +193,63 @@ def test_comet_pickle(tmpdir, monkeypatch):
     trainer2.logger.log_metrics({"acc": 1.0})
 
 
+def test_wandb_logger(tmpdir):
+    """Verify that basic functionality of wandb logger works."""
+    tutils.reset_seed()
+
+    try:
+        from pytorch_lightning.logging import WandbLogger
+    except ModuleNotFoundError:
+        return
+
+    hparams = tutils.get_hparams()
+    model = LightningTestModel(hparams)
+
+    wandb_dir = os.path.join(tmpdir, "wandb")
```

Comment:
I think that the sub folder is not needed, the temp dir is always clean and unique...